.gitignore

```diff
@@ -0,0 +1 @@
+flavor
```

Comment:
You can either move this into the `.gitignore` in the subdirectory, or provide `full/path/to/flavor`.